cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/module/slowquery/SlowQueryLogDecider.java

```diff
@@ -50,8 +50,7 @@ public boolean isForReporting(Query query) {
     private boolean executionTimeForLogging(long executionTimeInMilliseconds) {
         logger.trace("Checking if execution time:{} is above threshold: {}", executionTimeInMilliseconds,
                 slowQueryConfiguration.slowQueryThreshold());
-        if (slowQueryConfiguration.logAllQueries()
-                || executionTimeInMilliseconds >= slowQueryConfiguration.slowQueryThreshold()) {
+        if (executionTimeInMilliseconds > slowQueryConfiguration.slowQueryThreshold()) {
```

Comment:
Thats why I think the slow query threshold should be the first logged value instead of maximum allowed (greater or equals). With 0 you log all with 1 you log all that took 1 or more ms to execute.